app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -568,6 +568,8 @@ class HomeFragment : Fragment() {
         requireContext().settings().useNewTabTray.also {
             view?.add_tab_button?.isVisible = !it
             view?.tab_button?.isVisible = it
+            // Scrolls to the top of the screen
+            view?.homeAppBar?.setExpanded(true, false)
```

Comment:
Is this directive too broad?  Do we want to pass a flag so we aren't always assuming we should go to the top?

Reply:
@darkwing Not sure. I say lets land it and see what happens in Nightly 👍